Yank/commands/status.py

```diff
@@ -23,26 +27,104 @@
 YANK status
 
 Usage:
-  yank status (-s=STORE | --store=STORE) [-v | --verbose]
+  yank status (-y FILEPATH | --yaml=FILEPATH) [--status=STRING] [--njobs=INTEGER] [-v | --verbose]
 
 Description:
-  Get the current status of a running or completed simulation
+  Print the current status of the experiments.
 
 Required Arguments:
-  -s=STORE, --store=STORE       Storage directory for NetCDF data files.
+  -y, --yaml=FILEPATH           Path to the YAML script specifying options and/or how to
+                                set up and run the experiment.
 
 General Options:
-  -v, --verbose                 Print verbose output
+  --status=STRING               Print only the jobs in a particular status. Accepted values
+                                are "completed", "ongoing", and "pending". This works only
+                                if verbose is set.
+  --njobs=INTEGER               Print the job id associated to each experiment assuming
+                                njobs to be the one specified here.
+  -v, --verbose                 Print status of each experiment individually. If this is
+                                not set, only a summary of the status of all experiments
+                                is printed.
 
 """
 
+
 # =============================================================================================
 # COMMAND DISPATCH
 # =============================================================================================
 
+def find_contiguous_ids(job_ids):
+    """Return the contiguous job ids in the given list.
+
+    Returns
+    -------
+    contiguous_job_ids : str
+        The job ids organized in contiguous sets.
+
+    """
+    contiguous_job_ids = []
+    for k, g in itertools.groupby(enumerate(job_ids), lambda x:x[0]-x[1]):
+        group = list(map(operator.itemgetter(1), g))
+        if len(group) == 1:
+            contiguous_job_ids.append(str(group[0]))
+        else:
+            contiguous_job_ids.append('{}-{}'.format(group[0], group[-1]))
+    return ','.join(contiguous_job_ids)
+
 
 def dispatch(args):
```

Comment:
Can this function ever return a `False` status?

Reply:
Nope! This should always get dispatched since there are no subcommands.